tests/lukas.py

```diff
@@ -0,0 +1,96 @@
+import cipheydists
```

Comment:
this file isn't actually in enciphey, because you don't import it :P Please import :sparkles: